samples/python/21.corebot-app-insights/flight_booking_recognizer.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from botbuilder.ai.luis import LuisApplication, LuisRecognizer, LuisPredictionOptions
+from botbuilder.core import (
+    Recognizer,
+    RecognizerResult,
+    TurnContext,
+    BotTelemetryClient,
+    NullTelemetryClient,
+)
+
+from config import DefaultConfig
+
+
+class FlightBookingRecognizer(Recognizer):
+    def __init__(
+        self, configuration: DefaultConfig, telemetry_client: BotTelemetryClient = None
+    ):
+        self._recognizer = None
+
+        luis_is_configured = (
+            configuration.LUIS_APP_ID
+            and configuration.LUIS_API_KEY
+            and configuration.LUIS_API_HOST_NAME
+        )
+        if luis_is_configured:
+            # Set the recognizer options depending on which endpoint version you want to use e.g v2 or v3.
```

Comment:
@axelsrz The comments on lines 28-29 are missing in [13.corebot-app-insights/flight_booking_recognizer.py](https://github.com/microsoft/BotBuilder-Samples/blob/master/samples/python/13.core-bot/flight_booking_recognizer.py#L19). If they're relevant then they should be added to maintain parity between the samples.